doc/appendix-de-kassensichv/appendix-de-kassensichv.md

```diff
@@ -334,3 +334,38 @@ This table expands on the values provided in table [ftPayItemCase in General Par
 | `0x4445000000000016` | cash transfer from/to cash book <br /> not taxable <br /> DSFinV-K transformation required. negative amount gets converted to GV_TYP=Einzahlung. positive amount gets converted to GV_TYP=Auszahlung. amount=-amount.  in case of void-receipt everything turned | Keine | 1.3- |
 | `0x4445000000000017` | cash amount difference from/to till<br /> not taxable<br /> DSFinV-K transformation required. UST_Schluessel=5. GV_TYP=DifferenzSollIst. amount=-amount. | Keine | 1.3- |
 
+
+
+
+
+### Type of Signature: ftSignatureType
+
+The ftSignatureType indicates type and origin of the signature. The data type is `Int64` and can contain a country specific code which is a value following the ISO-3166-1-ALPHA-2 standard, converted from ASCII into hex and used as byte 8 and 7.
+
+For definitions regarding national laws, please refer to the appropriate appendix<span id="t-type-of-signature-ftsignaturetype-127">.</span>
+
+| **Value**            | **Description**          | **Service-Version** |
+|----------------------|--------------------------|---------------------|
+| `0x4445000000000001` | Signature according to kasssichv (qr-code content)                  | 1.3                 |
+| `0x4445000000000002` | Archiving required  | 1.3                |
+| `0x4445000000000003` | notification       | 1.3                 |
+| `0x4445000000000010` | start-transaction-result     | 1.3                 |
+| `0x4445000000000011` | finish-transaction-payload     | 1.3                 |
+| `0x4445000000000012` | finish-transaction-result     | 1.3                 |
+| `0x4445000000000013` | receipt / qr-version     | 1.3                 |
+| `0x4445000000000014` | receipt / kassenseriennummer     | 1.3                 |
+| `0x4445000000000015` | receipt / processtype     | 1.3                 |
+| `0x4445000000000016` | receipt / processdata     | 1.3                 |
+| `0x4445000000000017` | receipt / transaktionsnummer     | 1.3                 |
+| `0x4445000000000018` | receipt / signaturzähler     | 1.3                 |
+| `0x4445000000000019` | receipt / startzeit (start-transaction)    | 1.3                 |
+| `0x444500000000001A` | receipt / logtime     | 1.3                 |
+| `0x444500000000001B` | receipt / signaturalgorithmus     | 1.3                 |
+| `0x444500000000001C` | receipt / logtime-format     | 1.3                 |
+| `0x444500000000001D` | receipt / signatur     | 1.3                 |
```

Comment:
Should we differntiate between `signatur` (which in fact is the finish-transaction-signatur) and the signaturetype `finish-transaction-result 0x4445000000000012`

Reply:
`0x4445000000000012` is the result with date/time, signature-counter and signature-value (and signed hash-value) to give us the opportunity to work with this data also when it is not a pos-receipt. this data are in base64 format and are not printed to the receipt.

`0x444500000000001D` is the legal required line which needs to be printed at the receipt in case of a pos-receipt.